src/Orckestra.Composer/Services/ComposerContext.cs

```diff
@@ -193,10 +218,27 @@ public virtual string GetEncryptedCustomerId()
             return dto.EncryptedCustomerId;
         }
 
+
         /// <summary>
-        /// Initializes the customer identifier.
+        /// Initializes the scope item.
         /// </summary>
-        private void InitializeCustomerId()
+        private void InitializeScopeItem()
+        {
+            if (_scopeItem == null)
+            {
+                var p = new GetScopeParam
+                    {
+                        Scope = ScopeProvider.DefaultScope
```

Comment:
Can we use Scope property from this ComposerContext, instead of ScopeProvider.DefaultScope?